scripts/main.js

```diff
@@ -2,6 +2,14 @@ const mainForm = document.getElementById('main-form');
 const bookTitle = document.getElementById('book-title');
 const bookAuthor = document.getElementById('book-author');
 const bookList = document.querySelector('.book-list');
+const listA = document.getElementById('list-option');
+const newBookA = document.getElementById('new-book-option');
+const contactA = document.getElementById('contact-option');
+const logoA = document.getElementById('awesome-logo');
+const dateTime = document.getElementById('date-time');
+const d = new Date();
+const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+dateTime.innerHTML = `${months[d.getMonth()]} ${d.getDate()}th  ${d.getUTCFullYear()}, ${d.getHours()}:${d.getMinutes()}:${d.getSeconds()}`;
```

Comment:
- [ ] From the project specifications, you were supposed to use the [luxon ](https://moment.github.io/luxon/#/install) library to [get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Kindly work on this as it a hard requirement before your project can be approved